releasenotes.rb

```diff
@@ -3,9 +3,35 @@
 
 require "thor"
 require "git"
+require "octokit"
+require "logger"
 require "erb"
 require "ostruct"
 
+class GitHubCommitTags
+  attr_reader :client, :repository, :logger
+
+  def initialize(access_token:, repository:, logger: Logger.new(STDERR))
+    @client = Octokit::Client.new(access_token: access_token)
+    @repository = repository
+    @logger = logger
+  end
+
+  def special_deploy_requirements?(sha)
+    search_results = client.search_issues("#{sha} repo:#{repository} is:merged")
+
+    if search_results.total_count != 1
+      logger.warn("SHA #{sha} had #{search_results.total_count} issues, not just one; ignoring.")
```

Comment:
Got it!

I guess we can do the `> 1` change because this tool isn't responsible for keeping our process of merges ... and the message "had 0 issues, not just one; ignoring" is really strange and we know we have those single commits on the deploy branch.